lib/ansible/modules/identity/onepassword_info.py

```diff
@@ -90,27 +93,29 @@
 EXAMPLES = '''
 # Gather secrets from 1Password, assuming there is a 'password' field:
 - name: Get a password
-  onepassword_facts:
+  onepassword_info:
     search_terms: My 1Password item
   delegate_to: localhost
+  register_to: my_1password_item
```

Comment:
```suggestion
  register: my_1password_item
```
Does `register_to` actually work?

Reply:
(Same below)

Reply:
No, not sure what happened there 😂